README.md

```diff
@@ -6,27 +6,30 @@ Digital Forensics and Incident Response (DFIR) teams are groups of people in an
 
 ## Contents
 
-- [Adversary Emulation](#adversary-emulation)
-- [All in one tools](#all-in-one-tools)
-- [Books](#books)
-- [Communities](#communities)
-- [Disk Image Creation Tools](#disk-image-creation-tools)
-- [Evidence Collection](#evidence-collection)
-- [Incident Management](#incident-management)
-- [Linux Distributions](#linux-distributions)
-- [Linux Evidence Collection](#linux-evidence-collection)
-- [Log Analysis Tools](#log-analysis-tools)
-- [Memory Analysis Tools](#memory-analysis-tools)
-- [Memory Imaging Tools](#memory-imaging-tools)
-- [OSX Evidence Collection](#osx-evidence-collection)
-- [Other lists](#other-lists)
-- [Other tools](#other-tools)
-- [Playbooks](#playbooks)
-- [Process Dump Tools](#process-dump-tools)
-- [Sandboxing/reversing tools](#sandboxingreversing-tools)
-- [Timeline tools](#timeline-tools)
-- [Videos](#videos)
-- [Windows Evidence Collection](#windows-evidence-collection)
+- [Awesome Incident Response ![Awesome](https://github.com/sindresorhus/awesome)](#awesome-incident-response-img-src%22httpsgithubcomsindresorhusawesome%22-alt%22awesome%22)
```

Comment:
what is this change related to ?

Reply:
Looks like they accidentally deleted some lines by accident?

- [Contents] (#contents)
- [Books] (#books)	  
- [IR tools Collection] (#ir-tools-collection)